frontend/uv-poc/src/app/order/order.component.ts

```diff
@@ -15,6 +15,7 @@ export class OrderComponent implements OnInit {
   transactionId;
   errorMessage;
```

Comment:
Is this being used at all?

Reply:
This was lakshya component but i will still look into it